Database/Corrections/sodQuestFixes.lua

```diff
@@ -344,6 +344,7 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.objectivesText] = {"Bring symbols of the three owls to Loganaar in Moonglade. NOTE: To accept this quest, you need to bring at least one symbol, but you can also bring all three at once."},
         },
         [78261] = {
+            [questKeys.zoneOrSort] = zoneIDs.SHADOWFANG_KEEP,
```

Comment:
In the quest log, this quest is in the "Rogue" category. So we want to also show it as such.

```suggestion
            [questKeys.zoneOrSort] = sortKeys.ROGUE,
```

Reply:
It has to be `sortKeys`, not `zoneIDs`.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5430/commits/d1feee465f4669af7fb9cf052e43c65ae7369974

That was an oversight. Sorry about that! Should be good now.